### PR TITLE
Add com.xnview.XnViewMP (xnviewmp-flatpak)

### PR DESCRIPTION
**Pull request by scx**
**XnView MP** is the enhanced version of XnView Classic. It is a powerful picture viewer, browser and converter for Windows, Mac and Linux. This software can read more than 500 formats change picture size, reduce picture file size and much more!
https://github.com/scx/flathub/tree/com.xnview.XnViewMP

See also:
https://www.xnview.com/en/xnviewmp/
https://github.com/scx/xnviewmp-flatpak

### CODE REVIEW

**Comment by scx** [comment]
The vendor developer has been notified, but not answered yet.

**Comment by scx** [comment]
bot, build com.xnview.XnViewMP

**Comment by scx** [comment]
bot, build com.xnview.XnViewMP

**Comment by scx** [comment]
bot, build com.xnview.XnViewMP

**Comment by pierrexn** [comment]
Thanks to make a Flatpak package

File: XnViewMP.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
+name: XnViewMP
+buildsystem: simple
+sources:
+  - type: archive
```
Comment (barthalion) [inline XnViewMP.yaml]: It should use extra-data instead, as we're now indirectly violating their license by using test builds.

Comment (scx) [reply XnViewMP.yaml]: https://github.com/flathub/flathub/pull/883#discussion_r263404165

@alexlarsson said:
> Yeah, i think avoiding extra data when not needed due to licensing issues is a much better experience.

**Comment by scx** [comment]
bot, build com.xnview.XnViewMP

**Comment by scx** [comment]
bot, build com.xnview.XnViewMP

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/com.xnview.XnViewMP

**Comment by scx** [comment]
@nedrichards 

Thanks!

**Pull request closed without merging**